app/pages/lab-pages-editor/components/TasksPage/components/EditStepDialog/types/TextTask.jsx

```diff
@@ -81,14 +89,30 @@ function TextTask({
         </span>
       </div>
       <div className="input-row">
-        <label
-          className="big spacing-bottom-S"
-          htmlFor={`task-${taskKey}-help`}
-        >
-          Help Text
-        </label>
+        <div className="flex-row spacing-bottom-S">
```

Comment:
I'd strongly consider refactoring the shared code between DrawingTask / TextTask / QuestionTask into a file called `SharedTaskFields` or something like that. Without your context, I could imagine it being easy to change something in one and miss the others.